azure-pipelines.dfe.yml

```diff
@@ -275,6 +275,7 @@ jobs:
           Dockerfile: 'docker/Dockerfile.public-frontend'
           buildContext: '$(System.DefaultWorkingDirectory)'
           tags: $(Build.BuildNumber)
+          arguments: '--build-arg BUILD_BUILDNUMBER=$(Build.BuildNumber)'
```

Comment:
Is this actually sufficient? We need to convert the build argument into an environment variable inside of the image as well

Reply:
Ah good point, missed that part of the article. I've added this to the first stage of the docker image. Given that Next.js will transform this variable to `BUILD_NUMBER` at build time I think this should be sufficient (and we shouldn't need to include it in the second stage)

Reply:
Cool, yep this looks better now